scripts/clear-all-fault-leds.sh

```diff
@@ -68,11 +86,18 @@ then
         GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
         | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
-        echo "$line" | grep "powersupply\|dimm\|dcm\|fan" >/dev/null
+        echo "$line" | grep "powersupply\|dcm\|fan" >/dev/null
         rc=$?
         if [ $rc -eq 0 ]; then
             continue
         fi
+	echo "$line" | grep "dimm" >/dev/null
+	rc=$?
+	if [ $rc -eq 0 ]; then
+	    if [[ ${guardedDimms[@]} =~ $line ]]; then
+		continue
+	    fi
+	fi
         busctl set-property xyz.openbmc_project.Inventory.Manager "$line" xyz.openbmc_project.State.Decorator.OperationalStatus Functional b "$action";
     done
 else
```

Comment:
The changes need to be there in else part for excluded groups also

Reply:
Done